types/d3-dsv/index.d.ts

```diff
@@ -15,10 +15,9 @@
 
 /**
  * An object representing a DSV parsed row with values represented as strings.
- * When the DSV content is not well-structured and some column-values are missing, `undefined` is used as value.
  */
 export type DSVRowString<Columns extends string = string> = {
-    [key in Columns]: string | undefined;
+    [key in Columns]: string;
```

Comment:
Should `DSVRaw` be updated as well?

Reply:
Or should the `DSVRaw` type be removed altogether? I don't see any usages of it.

Reply:
At the least, it probably makes sense to remove `undefined` from it.

Reply:
I can't see what the purpose of `DSVRaw` is, but as this is my first pull request here I'm being cautious with my changes. Removing `DSVRaw` altogether or the `undefined` from it are both fine by me.

Reply:
Yeah, I'm not sure what the purpose of it is either, but it seems correlated with `DSVRowString`, so just removing the `undefined` seems like the pragmatic choice, if you don't mind doing that as part of this PR.